superset-frontend/cypress-base/cypress/e2e/explore/control.test.ts

```diff
@@ -157,107 +156,9 @@ describe('Test datatable', () => {
   });
 });
 
-describe('Time range filter', () => {
```

Comment:
Not sure yet if this one just got relocated, but it looks potentially important?

Reply:
This test was about the legacy time section that pre-dated generic x-axis.